File: ChainJava/src/chain/requests/DressingRoomRepairRequest.java
package chain.requests;

import java.util.Date;

/**
 * Created by Karolina_2 on 2015-04-01.
 */
public class DressingRoomRepairRequest extends Request {

    public DressingRoomRepairRequest(String title, Date date){
        this.title = title;
        this.date = date;
    }
}

File: ZTP_Zadanie3/ZTP_Zadanie3/chain/requests/DressingRoomRepairRequest.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace ZTP_Zadanie3.chain.wnioski
{
    class DressingRoomRepairRequest : Request
    {
        public DressingRoomRepairRequest(String title, DateTime date)
        {
            this.title = title;
            this.date = date;
        }
    }
}
